### PR TITLE
Add server.service.loadBalancerIP

### PR DESCRIPTION
**Pull request by mkuratczyk**
Useful to assign a static IP to the service

### CODE REVIEW

**Comment by rlex** [comment]
Hello, thanks for PR! 
server.service.loadBalancerIP needs to be added to values.yaml with comment, and helm-docs needs to be run to generate docs. 
Also, if you don't mind, please bump chart version in chart.yaml, so it will release right after merge

**Comment by rlex** [comment]
Thank you! Will merge now

File: charts/parca/templates/server-service.yaml [hunk 1]
```diff
@@ -11,6 +11,12 @@ metadata:
   {{- end }}
 spec:
   type: {{ .Values.server.service.type }}
+  {{- with .Values.server.service.loadBalancerIP }}
```
Comment (rlex) [inline charts/parca/templates/server-service.yaml:14]: uh, i actually missed that there is two declarations of same block? I will fix that in master today, i think.

Comment (mkuratczyk) [reply charts/parca/templates/server-service.yaml:14]: sorry for that 🤦

**Pull request merged**